src/modules/did/did.processor.ts

```diff
@@ -69,6 +69,10 @@ export class DIDProcessor {
       doc = await this.didService.incrementalRefreshCachedDocument(job.data);
     }
 
-    await this.pinQueue.add(PIN_CLAIM_JOB_NAME, doc);
+    for (const service of doc.service) {
+      await this.pinQueue.add(PIN_CLAIM_JOB_NAME, {
+        cid: service.serviceEndpoint,
+      });
+    }
```

Comment:
I would say that it's not important to change here, but curious if there is a reason that you choose a `for` loop over `map` & `Promise.all`?

Reply:
No reason. I just can't get used to Promise.all. I changed that